updateApi.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+BLUE='\033[1;34m'
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+NC='\033[0m'
+BOLD='\033[1m'
+
+set -e
+
+CURRENT_DIR=$(pwd)
+BASE_DIR=$(cd "$(dirname "$0")" && pwd)
+API_PKG="github.com/devfile/api/v2"
+SCHEMA_URL="https://raw.githubusercontent.com/devfile/api/master/schemas/latest/devfile.json"
+
+onError() {
+  cd "${CURRENT_DIR}"
+}
+trap 'onError' ERR
+
+
+echo "Updating devfile/api in go.mod"
+go get "${API_PKG}@master"
+
+echo "Get latest schema"
+schema=$(curl -L "${SCHEMA_URL}")
+
+#replace all ` with ' and write to schema file
+echo "Write to go file"
+go build *.go
+./main updateSchema "${schema}" "${SCHEMA_URL}"
```

Comment:
I think we can also put this in our github action, so that every PR is always updated and we dont need to kick off the script. 

If there is a breaking change, our tests should catch it. WDYT?

Reply:
I was thinking our put it in github action whenever the schema has been updated. However, some schema changes would need library change accordingly, or otherwise would cause failure. We need to discuss more on that if we want to use github action

Reply:
yeah i see your point, in that case; when the github action opens a PR and the test fails because the new schema introduced breaking change, we can manually update that PR opened by the bot 🤔